docs/content/about-iota/iota-architecture/iota-security.mdx

```diff
@@ -6,13 +6,15 @@ import {questions} from '../../../site/static/json/about-iota/iota-architecture/
 
 # Security
 
-## Assets
+## Access to Assets
 
-Security is one of the top priorities in IOTA. From the start, you need to authorize access to any asset using a private key known only to the asset owner. Additionally, the smart contract that defines the asset also sets its rules, which are further secured by the Move compiler to ensure assets are handled properly and no asset can be spent twice or lost due to a logic error. Once a transaction reaches finality, any assets that were modified or created will be updated, persisted, and available for further use. Even if an object is shared or immutable, you can still define access control logic in the smart contract that created it to ensure proper usage.
+IOTA uses two key mechanisms to secure access to assets. First, the asset owner must cryptographically prove knowledge of their private key tied to their account address to use or transfer an asset. Thus, account keys must be securely stored and used. Second, access to assets, which are represented as Move objects in a smart contract, follows the object ownership model of the Move language. The Move VM enforces ownership rules and ensures that no bytecode execution can violate them. The only potential risks stem from logic errors within smart contracts. Once a transaction reaches consensus and is committed to the ledger, any modified or created assets are persistently updated and available for further use.
 
 ## Validators and Consensus
```

Comment:
Move this section down, rename it to reflect consensus protocol, eg. "Validators and Consensus Protocol".